packages/@aws-cdk/aws-iam/lib/private/postprocess-policy-document.ts

```diff
@@ -84,9 +84,9 @@ export function normalizeStatement(s: StatementSchema) {
     Sid: _norm(s.Sid),
   });
 
-  function _norm(values: any, { unique }: { unique: boolean } = { unique: false }) {
+  function _norm(values: any, { unique = false }: { unique: boolean } = { unique: false }) {
```

Comment:
```suggestion
  function _norm(values: any, { unique = false }: { unique?: boolean } = {}) {
```

Reply:
Please. It's ugly, but also not worth talking about.